build.rs

```diff
@@ -1,18 +1,23 @@
 extern crate cc;
 
+use std::env::var;
+
 fn main() {
+	let target_os = var("CARGO_CFG_TARGET_OS").unwrap();
+	let target_family = var("CARGO_CFG_TARGET_FAMILY").unwrap();
+
 	let mut base_config = cc::Build::new();
 	base_config.include(".");
 	base_config.include("libusb/libusb");
 
-	if cfg!(target_os = "macos") {
+	if target_os == "macos" {
```

Comment:
Do these always match or should we use `contains` instead for all checks?

Reply:
I assume that `CARGO_CFG_TARGET_OS` contains the same value as `cfg!(target_os)`, and right now it works by matching exactly, so I suppose it will still continue to work by matching exactly.